### ISSUE TITLE
[dev-branch]-508-add landmarks to pages

### ISSUE THREAD

**Issue by Atticus29** [Symbiota/Symbiota#1045]
"There are no landmarks on the page such as banner, nav, and main. Landmarks allow blind users to navigate and find content quickly. Screen readers use ARIA landmarks to navigate a web page. If you use the U.S. Web Design System out of the box, it has all of that built in to help you. This issue carries across your site."

"There are no page headings either. Users of screen readers can use keyboard shortcuts to navigate directly to the first h1, which should be the main content of the web page. If there are no headings, a screen reader user must listen to more of the page to understand its structure. Visual users can take in much information about the page layout without having to read all the content. Blind users can’t do that."
